lib/chef/provider/group/usermod.rb

```diff
@@ -24,6 +24,8 @@ class Group
       class Usermod < Chef::Provider::Group::Groupadd
 
         provides :group, os: "openbsd"
+        provides :group, platform: %w(opensuse), platform_version: '>= 12.3'
+        provides :group, platform: %w(openindiana opensolaris nexentacore omnios solaris2 smartos hpux)
```

Comment:
Yeah, the old platform map was buggy, though, because it was based on platform and couldn't do platform_family or os.